cmd/kubeadm/app/phases/apiconfig/setupmaster.go

```diff
@@ -45,7 +45,13 @@ func attemptToUpdateMasterRoleLabelsAndTaints(client *clientset.Clientset) error
 		if n, err = client.Nodes().Get(node.GetHostname(""), metav1.GetOptions{}); err != nil {
```

Comment:
We could still log it of course using higher verbosity level, it may aid debugging in case if the request never succeeds...